test/taxonomic-units.js

```diff
@@ -14,7 +14,7 @@ describe('TaxonomicUnitWrapper', function () {
     it('should wrap a blank object', function () {
       const wrapper = new phyx.TaxonomicUnitWrapper({});
       assert.exists(wrapper);
-      assert.isUndefined(wrapper.label);
+      assert.equal(wrapper.label, 'Empty taxonomic unit');
```

Comment:
Well, here we go. Now we will have to change the value in two places if we wanted to change it in the future. Two is one too many.